app/uk/gov/hmrc/gform/controllers/ErrorHandler.scala

```diff
@@ -83,7 +86,10 @@ class ErrorHandler(
         case e @ UpstreamErrorResponse.WithStatusCode(statusCode) if statusCode == NotFound.intValue =>
           errResponder.notFound(requestHeader, e.message, maybeFormTemplate, smartUpstreamLogger)
         case e: NotFoundException => errResponder.notFound(requestHeader, e.message, maybeFormTemplate)
-        case e                    => errResponder.internalServerError(requestHeader, maybeFormTemplate, e)
+        case e: IllegalBindException =>
+          logger.info(s"URL: ${requestHeader.uri}, Query Params: ${requestHeader.queryString}")
```

Comment:
No need for a new `logger`, let use `smartUpstreamLogger.log` instead which will log message with ERROR level, which will help us to see it in Kibana